### ISSUE TITLE
FR: automatically show tooltips on elements that are focused with tab key

### ISSUE THREAD

**Issue by adidahiya** [palantir/blueprint#734]
When an element that is wrapped in a Tooltip is focused with <kbd>TAB</kbd>, it would be great if we could automatically trigger the tooltip to open and then close it when focus is lost. This would be a nice a11y enhancement. It should be configurable but enabled by default (opt-out behavior).
